Sources/PackageLoading/PackageBuilder.swift

```diff
@@ -137,49 +137,28 @@ extension Target {
     }
 }
 
-extension Target.Error: FixableError {
-    var error: String {
+extension Target.Error: CustomStringConvertible {
+    var description: String {
         switch self {
-          case .invalidName(let path, let name, let problem):
-            return "the directory \(path) has an invalid name ('\(name)'): \(problem.error)"
-          case .mixedSources(let path):
-            return "the target at \(path) contains mixed language source files"
+        case .invalidName(let path, let problem):
+            return "invalid target name at '\(path)'; \(problem)"
+        case .mixedSources(let path):
+            return "target at '\(path)' contains mixed language source files"
         case .duplicateTargets(let targets):
             return "duplicate targets found: " + targets.joined(separator: ", ")
         }
     }
-
-    var fix: String? {
```

Comment:
Here I don't agree. It think it is obvious for a user what to do in the case of the following errors:

* `invalid target name at 'path'; target names can not be empty`
* `invalid target name at 'path'; name of test targets must end in 'Tests'`
* `invalid target name at 'path'; name of non-test targets cannot end in 'Tests'`
* `target at 'path' contains mixed language source files`